assets/app/view/create_game.rb

```diff
@@ -11,6 +11,8 @@ class CreateGame < Form
     needs :num_players, default: 3, store: true
     needs :flash_opts, default: {}, store: true
     needs :user, default: nil, store: true
+    needs :access, default: :public, store: true
```

Comment:
it might be easier if we just inferred if the game was private or public based on the password, can we remove access?